hw/ip/rv_core_ibex/data/rv_core_ibex_testplan.hjson

```diff
@@ -0,0 +1,327 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Versio 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: "rv_core_ibex"
+  entries: [
+    {
+      name: arithmetic_basic_test
```

Comment:
Does the name need to match the test entry in the regression plan?

Reply:
No requirements exist, but I'll add back the `riscv_` prefix for clarity.

Reply:
The strings in the 'tests' array need to match the names in the regression plan.

Reply:
Done, thanks for the feedback!